### DISCUSSION TITLE
3.3.7 Release

### DISCUSSION THREAD

**Discussion by pgold** [bcb/jsonrpcclient#222] [General]
@bcb Would you be open to publishing a 3.3.7 release? There are a couple commits on the 3.3.x branch that are not included on 3.3.6. I understand that the library has moved on from the old 3.3.x model, but it could be helpful to some (surely it is to me) to be able to depend on a 3.3.7 release (instead of a patched version).

**Comment by bcb** [comment]
Yes sure, there's a `3.x` branch created for that purpose, (`main` is for 4.x). No reason why we can't release an update to version 3.
